### PR TITLE
fix(breadcrumb): accessibility enhancements

### PR DESCRIPTION
**Pull request by kumarivijeta**
## Summary

Added aria-label="Breadcrumb" & aria-current="page" for nav element and current page for Breadcrumb.

## ADO Story

https://dev.azure.com/Kyndryl/Shidoka%20-%20Bridge%20Design%20System/_workitems/edit/1942423

## Checklist

- [x] Used Conventional Commit messages as outlined in the contributing guide.
  - [ ] Noted breaking changes (if any).
- [ ] Documented/updated all props, events, slots, parts, etc with JSDoc.
  - [x] Ran the `analyze` command to update Storybook docs.
- [ ] Added/updated Stories with controls to cover all variants.
- [ ] Ran `test` locally to address any failures.
- [ ] Added/updated the Figma link for the Story's Design tab.
- [ ] Added any new component exports to the src/index.ts file

## Testing Instructions

Use voiceover for Testing

### CODE REVIEW

**Review by srpriyankashetty — APPROVED** [review]
Verified that screen reader read out the breadcrumb
Verified that once the page is loaded the screen reader reads the breadcrumb components.

image
image

File: src/components/reusable/breadcrumbs/breadcrumbs.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class Breadcrumbs extends LitElement {
 
   override render() {
     return html`
-      <nav class="breadcrumbs">
+      <nav aria-label="Breadcrumb" class="breadcrumbs">
```
Comment (brian-patrick-3) [inline src/components/reusable/breadcrumbs/breadcrumbs.ts:18]: The "Breadcrumbs" string cannot be hard-coded, it must be customizable via prop. Alternatively, application devs can place it directly on the kyn-breadcrumbs tag.

**Review by brian-patrick-3 — CHANGES_REQUESTED** [review]

File: src/components/reusable/breadcrumbs/breadcrumbs.ts (hunk 1 above)
Comment (kumarivijeta) [reply src/components/reusable/breadcrumbs/breadcrumbs.ts]: Addressed. Moved the aria-label to kyn-breadcrumbs tag.

**Review by brian-patrick-3 — APPROVED** [review]

**Pull request merged**